.lgtm.yml

```diff
@@ -5,19 +5,27 @@ path_classifiers:
     - cross-checks/c-checks/clang-plugin/test
     - cross-checks/rust-checks/rustc-plugin/tests
     - cross-checks/rust-checks/derive-macros/tests
+  library:
+    - dependencies
+    
 extraction:
   python:
     python_setup:
-      requirements_files:
-        - scripts/requirements.txt
       version: 3
       setup_py: false
   cpp:
-    # prepare:
+    prepare:
+      packages:
+        - python3-psutil
+        
     after_prepare:
-      - sudo scripts/provision_deb.sh
+      - "curl https://sh.rustup.rs -sSf | sh -s -- --no-modify-path -y"
+
+    configure:
+      command:
+        - "echo Deliberately not running cmake..."
```

Comment:
LGTM tries to automatically detect the build system. It finds some traces of cmake, so it tried to use that. Of course that's not the right way to build c2rust.